pypeman/remoteadmin.py

```diff
@@ -473,6 +491,12 @@ def do_list(self, channel, arg):
                 if arg == "--preview":
                     to_preview = True
                     args.remove(arg)
+                if arg.startswith("text="):
+                    text = arg.split("=")[1]
```

Comment:
`arg.split("=", 1)[1]` to allow having `=` in search text

Reply:
done